apps/emqx_bridge_s3tables/test/emqx_bridge_s3tables_SUITE.erl

```diff
@@ -1960,6 +1960,87 @@ t_parquet_3level_lists(TCConfig) when is_list(TCConfig) ->
     end),
     ok.
 
+-doc """
+Checks that we infer Access Key Id and Secret Access Key from Instance Metadata Service v2 APIs
+when those values are omitted.
+""".
+t_imdsv2_credentials(TCConfig0) when is_list(TCConfig0) ->
+    TCConfig = emqx_bridge_v2_testlib:proplist_update(
+        TCConfig0,
+        connector_config,
+        fun(Cfg0) ->
+            {_, Cfg1} = maps:take(<<"access_key_id">>, Cfg0),
+            {_, Cfg} = maps:take(<<"secret_access_key">>, Cfg1),
+            Cfg
```

Comment:
Is it basically this, or am I missing something?
```suggestion
            maps:without([<<"access_key_id">>, <<"secret_access_key">>], Cfg0)
```

Reply:
Not quite.  The current way is an assertion that the keys are there in the first place.